include/fiction/layouts/siqad_to_cartesian.hpp

```diff
@@ -0,0 +1,33 @@
+//
```

Comment:
What is the purpose of this file? Can it be deleted?

Reply:
This is an old file. I already deleted it yesterday

Reply:
Okay, then please also remove it from the git